procedures/observing_tools/Sky Almanac.ipynb

```diff
@@ -0,0 +1,697 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #27.        Get the airmass for a given target at a given location and at a given time.
</pre><p>Mention which model you're using to do this (plane parallel atmosphere)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>